.github/workflows/test.yml

```diff
@@ -0,0 +1,23 @@
+name: test
```

Comment:
why?

Reply:
you can set up your own fork for this purpose--just use the current one that runs when you PR into `main`

Reply:
I'm sorry I don't quite understand, it is added by my teammate, I'll delete this file and create another PR for this file later.